types/gridfs-stream/package.json

```
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "@types/mongodb": "^3.6.20"
+        "mongodb": "^4.4.1"
```

This was released today https://www.npmjs.com/package/mongodb/v/4.5.0
```suggestion
        "mongodb": "^4.5.0"
```